### PR TITLE
Add SteakHut Liquidity V2 on Avalanche

### PR DESCRIPTION
**Pull request by 0xSirloin**
Update of existing V1 pools to V2 pools on Avalanche

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/steakHut-lb exports TVL: 
        
 
 
    arbitrum                  13.30 k
    avax                      2.80 M
    
    total                    2.81 M

File: env.sample
Comment (g1nt0ki) [inline env.sample:1]: please revert this change

**Comment by llamatester** [comment]
The adapter at projects/steakHut-lb exports TVL: 
        
 
 
    avax-staking              637.77 k
    arbitrum                  13.32 k
    avax                      2.20 M
    staking                   637.77 k
    
    total                    2.21 M

**Comment by g1nt0ki** [comment]
okay, I have refactored the adapter a bit, we count own tokens under staking, that the my major change.

thanks for the PR
